### ISSUE TITLE
autocomplete improvement

### ISSUE THREAD

**Issue by NielsG** [arshad/Sketch-Navigator#1]
Woudl it be possible to have the autocomplete read partial artboard names? So let's say the artboard is called "iPhone/activities/post activity". Is it possible to autocomplete by just typing "post" or instance? This way the shortcut is kind of useless with longer path-like artboard names.
Cheers

**Comment by rorysmyth** [comment]
+1

**Comment by benjamindavid** [comment]
Agreed! It would also be nice if the artboard search was not case sensitive. Great plugin!
